http_check/datadog_checks/http_check/utils.py

```diff
@@ -2,6 +2,13 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import os
+import platform
+import sys
+
+EMBEDDED_DIR = 'embedded'
+
+if platform.system() == 'Windows':
```

Comment:
Any reason not using `datadog_checks.dev.utils.ON_WINDOWS` as in test ?

Reply:
ddev is not shipped with the Agent

Reply:
Oh right 👍 

Any reason not using https://github.com/DataDog/integrations-core/blob/d1547928ad0826e94fa06b02e0e7dd123752f124/datadog_checks_base/datadog_checks/base/utils/platform.py#L76-L77 😅

Reply:
I just hate the way the import looks 🙂 

Updated